styles/designs/carnival/parts/_lists-components.scss

```diff
@@ -206,39 +119,12 @@ $UnorderedList__Item--AppendixGlossary: (
   )
 );
 
-// Exercises
-$UnorderedList--FirstLevel--Exercise: (
-    _name: "ExerciseFirstLevelUnorderedList",
-    _subselector: " > ul",
-    _properties: (
-        margin-left: h-spacing(3),
-        margin-bottom: v-spacing(1),
-    )
-);
 
-$OrderedList--FirstLevel--Exercise: (
-    _name: "ExerciseFirstLevelOrderedList",
-    _subselector: " > ol",
```

Comment:
Lists created separately for exercises previously has been removed due to standardization (lists in the rest of the themes don't have bottom margins).